tools/aequatus/aequatus.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="aequatus-generator" name="Aequatus generator" version="0.1.1">
+    <description>Aquatus generator takes tree, gene and alignment information and join them in as aequatus format.</description>
```

Comment:
Remove "Aquatus generator ", the description is appended to the name in the tool panel.